bubuku/features/rolling_restart.py

```diff
@@ -0,0 +1,214 @@
+import logging
+from time import time
+
+import requests
+
+from bubuku.api import ApiConfig
+from bubuku.aws import AWSResources, node, volume
+from bubuku.aws.cluster_config import ClusterConfig
+from bubuku.aws.ec2_node import EC2
+from bubuku.controller import Change
+from bubuku.zookeeper import BukuExhibitor
+
+_LOG = logging.getLogger('bubuku.features.rolling_restart')
+
+
+class RollingRestartChange(Change):
+    def __init__(self, zk: BukuExhibitor, cluster_config: ClusterConfig,
+                 broker_id_to_restart: str,
+                 image: str,
+                 instance_type: str,
+                 scalyr_key: str,
+                 scalyr_region: str,
+                 kms_key_id: str):
+        self.zk = zk
+        self.cluster_config = cluster_config
+        self.cluster_config.set_application_version(image)
+        self.cluster_config.set_instance_type(instance_type)
+        self.cluster_config.set_scalyr_account_key(scalyr_key)
+        self.cluster_config.set_scalyr_region(scalyr_region)
+        self.cluster_config.set_kms_key_id(kms_key_id)
+        self.state_context = StateContext(self.zk, self.cluster_config, broker_id_to_restart)
+
+    def get_name(self) -> str:
+        return 'rolling_restart'
+
+    def can_run(self, current_actions):
```

Comment:
We should also probably add `rolling-restart` to `can_run` and `should_be_paused` checks in [`BaseRebalanceChange.py`](https://github.com/zalando-nakadi/bubuku/blob/master/bubuku/features/rebalance/__init__.py), to avoid starting rebalance during a restart. Right?